templates/python/custom-copilot-basic/env/.env.testtool.user.tpl

```diff
@@ -0,0 +1,13 @@
+# This file includes environment variables that will not be committed to git by default. You can set these environment variables in your CI/CD system for your project.
+
+# If you're adding a secret value, add SECRET_ prefix to the name so Teams Toolkit can handle them properly
+# Secrets. Keys prefixed with `SECRET_` will be masked in Teams Toolkit logs.
+{{#useOpenAI}}
+SECRET_OPENAI_API_KEY=<openai-api-key>
```

Comment:
If users fill in the key/endpoint when scaffolding, we should put the value here.

Reply:
will update it